lib/thumbnex.ex

```diff
@@ -23,31 +25,41 @@ defmodule Thumbnex do
   * `:time_offset` - Timestamp in seconds at which to take screenshot, for videos and GIFs.
     By default picks a time near the beginning, based on video duration.
   """
-  @spec create_thumbnail(binary, binary, Keyword.t) :: :ok
+  @spec create_thumbnail(binary, binary, Keyword.t()) ::
+          {:ok, nil} | {:error, any()}
```

Comment:
I'd prefer to keep the `:ok` instead of tuple. That can change in the future if more information is returned from the function, but not as part of this pull request. Thanks.

Reply:
fixed